src/io/flutter/FlutterInitializer.java

```diff
@@ -113,6 +114,16 @@ public static void setVerboseLogging(boolean value) {
     properties.setValue(verboseLoggingKey, value);
   }
 
+  public static boolean isMemoryDashboard() {
```

Comment:
From discussion, I think we can remove the user preference, and always have the button visible.